lib/kitchen/docker/helpers/image_helper.rb

```diff
@@ -35,7 +35,7 @@ def parse_image_id(output)
 
         def remove_image(state)
           image_id = state[:image_id]
-          docker_command("rmi #{image_id}")
+          run_command("docker ps -a | grep -q #{image_id} && echo 'Can not delete such image; It is being used by running container/s' || docker rmi #{image_id}")
```

Comment:
Could we implement this using a new method inside this helper instead of in line with bash? I'd rather raise an exception and allow test kitchen to handle the exception instead of writing a message to stdout.

Something more like this?
```ruby
def image_in_use?
  # determine if image is in use with a running container
end

def remove_image(state)
  image_id = state[:image_id]
  raise "Can not delete image #{state[:image_id]}. It is being used by a running container." if image_in_use?
  docker_command("rmi #{image_id}")
end
```